examples/experimental/openmmforcefields/gaff.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
<p>This prose mentions Interchange combination, and I think that's the use case that most people coming to this notebook have in mind. Since the notebook already has warning lights from the INTERCHANGE_EXPERIMENTAL flag and the large number of private methods used, it may be worthwhile to put in a simple combination here (even just making a second copy of the molecule translated a bit and then combining it with the first, even if there's no reference energy to match to).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-interchange/pull/857/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good idea ... let's see if turning this molecule into a dimer can lead to a nice visualization, as well

Reply:
Visualization is blocked by #858